wp-content/themes/oceanwp-child/single-initiative.php

```diff
@@ -191,22 +191,22 @@
 								<div class="col-md-8 col-lg-8 col-sm-12">
 									<div class="row">
 						<?php 
-							$the_post_id = get_the_ID();
-							$actions = wp_get_post_terms($the_post_id, 'actiontype', ['']);
-							
-							if(empty($actions) || ! is_array($actions)){
-								echo "No Actions";
+							$initiative_actiontypes = get_post_meta($the_post_id, 'initiative');
```

Comment:
Change variable to `$initiative_actions_types`

Reply:
Done